### PR TITLE
fix: (platform) addressed list header label issue

### PR DESCRIPTION
**Pull request by sKudum**
#### Please provide a link to the associated issue.
 #3423 
 #3266 
 #3416 

#### Please provide a brief summary of this pull request.

labels with special characters in toolbar are not supported in rtl to ltr mode.  So removed special charters in the label text as per current toolbar styles support.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by JKMarkowski** [comment]
LGTM, but could you wait until #3400 is merged? So you would be able to just put `[glyph]=""` to `fd-button`

**Comment by sKudum** [comment]
> LGTM, but could you wait until #3400 is merged? So you would be able to just put `[glyph]=""` to `fd-button`

Hi @JKMarkowski , Updated with 3400.

**Review by KevinOkamoto — APPROVED** [review]
Looks good.

**Review by JKMarkowski — APPROVED** [review]

**Pull request merged**
